src/tests/verify_resources.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import collections
+import glob
+import yaml
+import sys
+import os
+import re
+
+_topdir = os.path.abspath(os.path.dirname(__file__) + "/../..")
+
+class autodict(collections.defaultdict):
+    def __init__(self,*other):
+        collections.defaultdict.__init__(self, self.__class__, *other)
+    def __add__ (self, other):
+        return other
+    def __repr__(self):
+        return '%s(%s)' % (self.__class__.__name__, dict.__repr__(self))
+
+def rgfilter(fn):
+    return not (fn.endswith("_downtime.yaml") or fn.endswith("/SITE.yaml"))
+
+def rgname(fn):
+    return re.search(r'/([^/]+)\.yaml$', fn).group(1)
+
+def sumvals(d):
+    return sum(list(zip(*d.items()))[1]) if d else 0
```

Comment:
What does this do?

Reply:
I don't know what possessed me to write it that way, but it simply sums the values of a dictionary `d`.  So the name is pretty appropriate.  If i were to add a comment it would just be, `# sum dict values`, and i would not take any further time to elaborate on how python's `zip` builtin function works.

But on second glance there's a much simpler way to implement this, which i'll do instead.